### DISCUSSION TITLE
May I clear cache folder after create sectors?

### DISCUSSION THREAD

**Discussion by seantw1974** [Chia-Network/chia-blockchain#2029] [Q&A]
I found my temp cache folder still leave lots of files ...
May I delete them ?

And also got an swap file at dest folder ? Have to keep ?

**Comment by seantw1974** [comment]
May I delete files in Temporary Directory after plot ?

**Reply by dorkmo** [reply]
temporary directory should be empty after each plot is finished, if there is data in there it is likely from when plotting was stopped partly through. its best to delete anythign in the temp directory after a failure
